R-package/R/xgb.config.R

```diff
@@ -4,7 +4,17 @@
 #' values of one or more global-scope parameters. Use \code{xgb.get.config} to fetch the current
 #' values of all global-scope parameters (listed in
 #' \url{https://xgboost.readthedocs.io/en/stable/parameter.html}).
+#' @details
+#' Note that some functions might use a globally-configured number of threads, which is not
+#' managed by this configuration. Typically, XGBoost methods accept an `nthreads` parameter
+#' controlling the number of parallel  threads that will be used, but some functionalities
+#' might execute functions before this parameter can be supplied.
```

Comment:
The only feature that has problems with local `nthread` is the model serialization, let's be more specific here to reduce confusion.

Reply:
Updated.